tfe/resource_tfe_workspace.go

```diff
@@ -55,9 +57,23 @@ func resourceTFEWorkspace() *schema.Resource {
 			},
 
 			"operations": {
-				Type:     schema.TypeBool,
-				Optional: true,
-				Default:  true,
+				Type:          schema.TypeBool,
+				Optional:      true,
+				Computed:      true,
+				ConflictsWith: []string{"execution_mode", "agent_pool_id"},
+			},
+
+			"execution_mode": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				Computed:      true,
+				ConflictsWith: []string{"operations"},
+			},
+
+			"agent_pool_id": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				ConflictsWith: []string{"operations"},
```

Comment:
nit: we generally try to keep these in alphabetical order after the most important identifying attributes (like name and organization). so agent_pool_id should be up above auto_apply and execution_mode should be above file_triggers_enabled

Reply:
Huh. I kept these close to `operations` intentionally but that sounds good to me 👍🏻